src/app/core/models/part.spec.ts

```diff
@@ -0,0 +1,209 @@
+
+                        expect(result).toBe(playedDuration)
+                    })
+                })
+            })
+
+            describe('part has manual take next', () => {
+                describe('part has expected duration', () => {
+                    describe('expected duration is longer than the time since part was taken', () => {
+                        it('returns the expected duration', () => {
+                            const expectedDuration: number = 3456
+                            const playedDuration: number = 2000
+                            const testee = createPart({
+                                isOnAir: true,
+                                expectedDuration,
+                                executedAt: Date.now()
+                            })
+                            jasmine.clock().tick(playedDuration)
+
+                            const result: number = testee.getDuration()
+
+                            expect(result).toBe(expectedDuration)
+                        })
+                    })
+
+                    describe('expected duration is shorter than the time since part was taken', () => {
+                        it('returns the time since part was taken', () => {
+                            const expectedDuration: number = 3456
+                            const playedDuration: number = 4000
+                            const testee = createPart({
+                                isOnAir: true,
+                                expectedDuration,
+                                executedAt: Date.now()
+                            })
+                            jasmine.clock().tick(playedDuration)
+
+                            const result: number = testee.getDuration()
+
+                            expect(result).toBe(playedDuration)
+                        })
+                    })
+                })
+
+                describe('part has no expected duration', () => {
+                    describe('default part duration is longer than the time since part was taken', () => {
+                        it('returns the default part duration', () => {
+                            const playedDuration: number = 3000
+                            const testee = createPart({
+                                isOnAir: true,
+                                executedAt: Date.now()
+                            })
+                            jasmine.clock().tick(playedDuration)
+
+                            const result: number = testee.getDuration()
+
+                            expect(result).toBe(DEFAULT_PART_DURATION_IN_MS)
+                        })
+                    })
+
+                    describe('default part duration is shorter than the time since part was taken', () => {
+                        it('returns the time since part was taken', () => {
+                            const playedDuration: number = 5000
```

Comment:
Maybe do: `const playedDuration: number = DEFAULT_PART_DURATION_IN_MS + 5000`?
Then if DEFAULT_PART_DURATION_IN_MS has its value updated in the future to something higher than 5000, then this test wont break.

Reply:
Good observation. Is changed now.